Assets/LoggingManager/LogStore.cs

```diff
@@ -0,0 +1,276 @@
+﻿using System;
+using System.CodeDom;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Boo.Lang.Runtime;
+using UnityEngine;
+
+
+public enum LogType
+{
+    Normal,
+    Meta
+}
+
+public class LogStore : MonoBehaviour
+{
+
+    private SortedDictionary<string, List<string>> logs;
+    private StringBuilder logString;
+    public string Label { get; set; }
+
+    private LogType logType;
+    private bool logCommonColumns;
+
+    private int nbLines;
```

Comment:
nb stands for "Number" (in french we usually use this abbreviation 🥖) , so this variable means number of lines. I think i'll rename it "linesCount", it will be more understandable.

Reply:
@Vincent-SD  alternatively, we should call it "rowCount" - in line with EndRow, currentRowLogs etc.